knowledge-base/troubleshoot-report-server.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Troubleshooting Report Server
+description: Diagnose problems with the Telerik Report Server application
+type: troubleshooting
+page_title: Troubleshoot Report Server Manager
+slug: troubleshoot-report-server
+position: 
+tags: 
+ticketid: 1400116
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Report Server</td>
+	</tr>
+</table>
+
+
+## Description
+Sometimes it is possible to experience problems with the Report Server the reasons for which are not apparent. For example, restricted permission for accessing the File Storage, Internal Server Errors due to unaccessible Storage or missing/corrupted report resources (e.g. invalid image), etc.
+
+## Solution
+We recommend using the following basic approaches for troubleshooting :
+  
+1. Check whether there are any errors in the browser console.  
+
+2. Use [Fiddler ](https://www.telerik.com/download/fiddler) to see the exact requests from the Report Manager Viewer to the server, and the corresponding responses - they may indicate what has failed.  
+
+3. Attach a [Trace Listener](https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/how-to-create-and-initialize-trace-listeners) to the Report Server. The generated log should contain the entire Stack Trace of the error causing the failure. This can be done in the _Web.config_ file of the Report Server application. By default, it is in the (_Telerik Report Server installation folder_)\\Telerik.ReportServer.Web (e.g. _C:\\Program Files (x86)\\Progress\\Telerik Report Server\\Telerik.ReportServer.Web_). The code for enabling the _Trace Listener_ may look like :  
```

Comment:
What about a trace listener in the scheduling service?

Reply:
To be included

Reply:
Done